modules/custom-css/custom-css-4.7.php

```diff
@@ -0,0 +1,781 @@
+		add_filter( 'customize_update_custom_css_post_content_args', array( __CLASS__, 'customize_update_custom_css_post_content_args' ), 10, 3 );
+		add_filter( 'update_custom_css_data', array( __CLASS__, 'update_custom_css_data' ), 10, 2 );
+
+		// Handle Sass/LESS
+		add_filter( 'customize_value_custom_css', array( __CLASS__, 'customize_value_custom_css' ), 10, 2 );
+		add_filter( 'customize_update_custom_css_post_content_args', array( __CLASS__, 'customize_update_custom_css_post_content_args' ), 10, 3 );
+
+		// Stuff for stripping out the theme's default stylesheet...
+		add_filter( 'stylesheet_uri', array( __CLASS__, 'style_filter' ) );
+		add_filter( 'safecss_skip_stylesheet', array( __CLASS__, 'preview_skip_stylesheet' ) );
+
+		// Stuff for overriding content width...
+		add_action( 'customize_preview_init', array( __CLASS__, 'preview_content_width' ) );
+		add_filter( 'jetpack_content_width', array( __CLASS__, 'jetpack_content_width' ) );
+		add_filter( 'editor_max_image_size', array( __CLASS__, 'editor_max_image_size' ), 10, 3 );
+		add_action( 'template_redirect', array( __CLASS__, 'set_content_width' ) );
+		add_action( 'admin_init', array( __CLASS__, 'set_content_width' ) );
+
+		// Stuff?
+	}
+
+	public static function init() {
+		$min = '.min';
+		if ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) {
+			$min = '';
+		}
+
+		wp_register_style( 'jetpack-codemirror',      plugins_url( "custom-css/css/codemirror.css", __FILE__ ), array(), '20120905' );
+		wp_register_style( 'jetpack-customizer-css',  plugins_url( 'custom-css/css/customizer-control.css', __FILE__ ), array( 'jetpack-codemirror' ), '20140728' );
+		wp_register_script( 'jetpack-codemirror',     plugins_url( "custom-css/js/codemirror.min.js", __FILE__ ), array(), '3.16', true );
+		wp_register_script( 'jetpack-customizer-css', plugins_url( 'custom-css/js/core-customizer-css.js', __FILE__ ), array(  'customize-controls', 'underscore', 'jetpack-codemirror' ), JETPACK__VERSION, true );
+
+		wp_register_script( 'jetpack-customizer-css-preview', plugins_url( 'custom-css/js/core-customizer-css-preview.js', __FILE__ ), array( 'customize-selective-refresh' ), JETPACK__VERSION, true );
+	}
+
+	public static function customize_preview_init() {
+		add_filter( 'wp_get_custom_css', array( __CLASS__, 'customize_preview_wp_get_custom_css' ) );
+	}
+
+	public static function map_meta_cap( $caps, $cap ) {
+		if ( 'edit_css' === $cap ) {
+			$caps = array( 'edit_theme_options' );
+		}
+		return $caps;
+	}
+
+	public static function admin_menu() {
+		// Add in our legacy page to support old bookmarks and such.
+		add_theme_page( __( 'CSS', 'jetpack' ), __( 'Edit CSS', 'jetpack' ), 'edit_theme_options', 'editcss', array( __CLASS__, 'admin_page' ) );
+
+		// Add in our new page slug that will redirect to the customizer.
+		$hook = add_theme_page( __( 'CSS', 'jetpack' ), __( 'Edit CSS ⚡', 'jetpack' ), 'edit_theme_options', 'editcss-customizer-redirect', array( __CLASS__, 'admin_page' ) );
```

Comment:
Will we eventually only have the regular "Edit CSS" menu item (without the ⚡ emoji), and have it point to `editcss-customizer-redirect`?